docs/marketplace/dev/module-migration-guides/migration-guide-productmerchantportalgui.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Migration guide - ProductMerchantPortalGui
+description: Use the guide to update versions to the newer ones of the ProductMerchantPortalGui module.
+template: module-migration-guide-template
+---
+
+## Upgrading from version 1.* to version 2.0.0
+
+In this new version of the **ProductMerchantPortalGui** module, we have added support product approval feature. You can find more details about the changes on the [ProductMerchantPortalGui module](https://github.com/spryker/product-merchant-portal-gui/releases) release page.
```

Comment:
You can find more details about the changes on the [ProductMerchantPortalGui release](https://github.com/spryker/product-merchant-portal-gui/releases) page.